dnsimple/domains_email_forwards.go

```diff
@@ -7,12 +7,17 @@ import (
 
 // EmailForward represents an email forward in DNSimple.
 type EmailForward struct {
-	ID        int64  `json:"id,omitempty"`
-	DomainID  int64  `json:"domain_id,omitempty"`
+	ID       int64 `json:"id,omitempty"`
+	DomainID int64 `json:"domain_id,omitempty"`
+	// For requests, this field is deprecated, please use `AliasName` instead.
 	From      string `json:"from,omitempty"`
-	To        string `json:"to,omitempty"`
-	CreatedAt string `json:"created_at,omitempty"`
-	UpdatedAt string `json:"updated_at,omitempty"`
+	// WARNING: This is not set in responses, please use `From` instead.
```

Comment:
This instead would be:

```suggestion
	// WARNING: This is not set in responses, please use `AliasEmail` instead.
```